server/migrations/env.py

```
@@ -0,0 +1,57 @@
+from sqlalchemy import engine_from_config, pool
+from alembic import context
+
+# Interpret the config file for Python logging.
+config = context.config
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+# Use app settings to build the URL at runtime
+try:
+    from app.core.settings import get_settings
+except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)